### ISSUE TITLE
google_compute_instance_group_manager shows a non-existing diff

### ISSUE THREAD

**Issue by dimisjim** [infracost/infracost#3241]
Hello,

I am trying infracost out.

Overall it looks like it works as intended, however there is a strange outputted in every PR:

```
Project: atlantis-main
Module path: atlantis

~ module.atlantis.google_compute_instance_group_manager.default
  $0.00 ($93 → $93)

    ~ SSD provisioned storage (pd-ssd)
      -$9 ($2 → $2)

Monthly cost change for atlantis-main (Module path: atlantis)
Amount:  $0.00 ($109 → $109)
Percent: 0%
```

For reference this is what we use for the diff calculations:
```
- name: Generate Infracost cost estimate baseline
        run: |
          infracost breakdown \
                    --config-file infracost.yaml \
                    --format=json \
                    --out-file=/tmp/infracost-base.json

      - name: Generate Infracost diff
        run: |
          infracost diff \
                    --config-file infracost.yaml \
                    --format=json \
                    --compare-to=/tmp/infracost-base.json \
                    --out-file=/tmp/infracost.json

      - name: Post Infracost comment
        run: |
            infracost comment github --path=/tmp/infracost.json \
                      --repo=$GITHUB_REPOSITORY \
                      --github-token=${{ github.token }} \
                      --pull-request=${{ github.event.pull_request.number }} \
                      --behavior=update
```

I would expect this to not be outputted at all as the diff is 0

**Comment by aliscott** [comment]
@dimisjim are you able to share the `/tmp/infracost.json` file?

**Comment by dimisjim** [comment]
@aliscott by defacing some values there, sure, however it's a huge file

are you interested in any particular section?

for example this is the diff section in the json for the resource being outputted as a diff:
```
"diff": {
  "resources": [
    {
      "name": "module.atlantis.google_compute_instance_group_manager.default",
      "resourceType": "google_compute_instance_group_manager",
      "defaultTags": {},
      "metadata": {},
      "hourlyCost": "0",
      "monthlyCost": "0",
      "monthlyUsageCost": "0",
      "costComponents": [
        {
          "name": "SSD provisioned storage (pd-ssd)",
          "unit": "GB",
          "hourlyQuantity": "-0.0547945205479452",
          "monthlyQuantity": "-40",
          "price": "0",
          "hourlyCost": "-0.0121095890410958892",
          "monthlyCost": "-8.84",
          "priceNotFound": false
        }
      ]
    }
  ],
  "totalHourlyCost": "0",
  "totalMonthlyCost": "0",
  "totalMonthlyUsageCost": "0"
},
```

**Comment by dimisjim** [comment]
@aliscott it's the one from here: https://github.com/runatlantis/terraform-gce-atlantis/blob/f80cc5be5f181bd6dfd759769dacf7999ab3dca8/main.tf#L252

**Comment by dimisjim** [comment]
Hey @aliscott 

When is the next release planned?

**Comment by aliscott** [comment]
@dimisjim this has been released in [v0.10.40](https://github.com/infracost/infracost/releases/tag/v0.10.40) 🙏
